modules/mobile/spec/request/contact_info_request_spec.rb

```diff
@@ -0,0 +1,188 @@
+      'effectiveEndDate' => nil,
+      'effectiveStartDate' => nil,
+      'geocodeDate' => '2018-04-13T17:01:18.000Z',
+      'geocodePrecision' => 100.0,
+      'id' => 124,
+      'internationalPostalCode' => nil,
+      'latitude' => 38.901,
+      'longitude' => -77.0347,
+      'province' => nil,
+      'sourceDate' => '2018-04-09T11:52:03.000-06:00',
+      'sourceSystemUser' => nil,
+      'stateCode' => 'DC',
+      'updatedAt' => '2017-04-09T11:52:03.000-06:00',
+      'validationKey' => nil,
+      'vet360Id' => '12345',
+      'zipCode' => '20011',
+      'zipCodeSuffix' => nil,
+      'badAddress' => true
+
+    }
+  end
+
+  let(:home_phone) do
+    {
+      'areaCode' => '303',
+      'countryCode' => '1',
+      'createdAt' => '2017-04-09T11:52:03.000-06:00',
+      'extension' => nil,
+      'effectiveEndDate' => nil,
+      'effectiveStartDate' => nil,
+      'id' => 789,
+      'isInternational' => false,
+      'isTextable' => false,
+      'isTextPermitted' => false,
+      'isTty' => true,
+      'isVoicemailable' => true,
+      'phoneNumber' => '5551234',
+      'phoneType' => 'HOME',
+      'sourceDate' => '2018-04-09T11:52:03.000-06:00',
+      'sourceSystemUser' => nil,
+      'updatedAt' => '2017-04-09T11:52:03.000-06:00',
+      'vet360Id' => '12345'
+    }
+  end
+
+  let(:mobile_phone) do
+    {
+      'areaCode' => '303',
+      'countryCode' => '1',
+      'createdAt' => '2017-04-09T11:52:03.000-06:00',
+      'extension' => nil,
+      'effectiveEndDate' => nil,
+      'effectiveStartDate' => nil,
+      'id' => 790,
+      'isInternational' => false,
+      'isTextable' => false,
+      'isTextPermitted' => false,
+      'isTty' => true,
+      'isVoicemailable' => true,
+      'phoneNumber' => '5551234',
+      'phoneType' => 'MOBILE',
+      'sourceDate' => '2018-04-09T11:52:03.000-06:00',
+      'sourceSystemUser' => nil,
+      'updatedAt' => '2017-04-09T11:52:03.000-06:00',
+      'vet360Id' => '12345'
+    }
+  end
+
+  let(:work_phone) do
+    {
+      'areaCode' => '303',
+      'countryCode' => '1',
+      'createdAt' => '2017-04-09T11:52:03.000-06:00',
+      'extension' => nil,
+      'effectiveEndDate' => nil,
+      'effectiveStartDate' => nil,
+      'id' => 791,
+      'isInternational' => false,
+      'isTextable' => false,
+      'isTextPermitted' => false,
+      'isTty' => true,
+      'isVoicemailable' => true,
+      'phoneNumber' => '5551234',
+      'phoneType' => 'WORK',
+      'sourceDate' => '2018-04-09T11:52:03.000-06:00',
+      'sourceSystemUser' => nil,
+      'updatedAt' => '2017-04-09T11:52:03.000-06:00',
+      'vet360Id' => '12345'
+    }
+  end
+
+  before do
+    iam_sign_in(FactoryBot.build(:iam_user))
+  end
+
+  describe 'GET /mobile/v0/user/contact_info with vet360 id' do
+    context 'valid user' do
+      before do
+        get('/mobile/v0/user/contact_info', headers: iam_headers)
+      end
+
+      it 'returns full contact information' do
```

Comment:
I think we should also test for user id. I think that would have revealed the fact that we provided the entire user object and not just the id.